### PR TITLE
[master] Removing FragmentedChangePitStop [6891]

### PR DESCRIPTION
**Pull request by IkerLuengo**
Port of #846

### CODE REVIEW

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Linux/958/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/797/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Windows/955/)
* Performance Build Status

File: include/fastdds/rtps/common/CacheChange.h [hunk 1]
```diff
@@ -136,7 +136,7 @@ struct RTPS_DllAPI CacheChange_t
         serializedPayload.encapsulation = ch_ptr->serializedPayload.encapsulation;
 
         // Copy fragment size and initialize missing list to 'all fragments missing'
-        setFragmentSize(ch_ptr->fragment_size_, true);
+        setFragmentSize(ch_ptr->fragment_size_, false);
```
Comment (MiguelCompany) [inline include/fastdds/rtps/common/CacheChange.h]: May be a good idea to change the comment on the previous line to something like `// Just copy fragment size and calculate fragment count`

Comment (IkerLuengo) [reply include/fastdds/rtps/common/CacheChange.h]: OK

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Linux/990/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/818/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Windows/986/)
* Performance [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Performance%20Tests%20Localhost/295/)

**Comment by MiguelCompany** [comment]
CI glitch on Mac. Relaunched:
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/820/)

**Comment by MiguelCompany** [comment]
Mac reported `100% tests passed, 0 tests failed out of 647`, but then marked build as failed

**Pull request merged**